app/helpers/alchemy/admin/elements_helper.rb

```diff
@@ -54,30 +54,42 @@ def elements_for_select(elements)
         end
       end
 
-      # Returns all elements that could be placed on that page because of the pages layout.
+      # Returns all elements that can be placed on the current page.
       # The elements will be grouped by cell.
+      #
+      # @param [Array] elements
+      #   collection of element objects
+      # @param [String] object_method
+      #   method that is called on the element objects used for the select option value
+      #
       def grouped_elements_for_select(elements, object_method = 'name')
         return "" if elements.blank?
         cells_definition = @page.cell_definitions
         return "" if cells_definition.blank?
```

Comment:
We should return an empty `Array`, because the method returns an `Array` anyway.